mcs/tools/upload-to-sentry/Makefile

```diff
@@ -0,0 +1,41 @@
+thisdir = tools/upload-to-sentry
+SUBDIRS =
+include ../../build/rules.make
+
+PROGRAM = upload-to-sentry.exe
+LIB_REFS = \
+	System \
+	System.Core \
+	System.Data \
+	System.Runtime.Serialization \
+	System.Xml \
+	System.Xml.Linq \
+	Mono.Posix
+NO_INSTALL = yes
+
+SENTRY_ROOT := $(shell dirname "$(topdir)/../")
+# These come from the environment
+SENTRY_ENV_OPTIONS = MONO_SENTRY_ROOT="$(SENTRY_ROOT)" MONO_SENTRY_OS="$(MONO_SENTRY_OS)" MONO_SENTRY_URL="$(MONO_SENTRY_URL)" 
+
+upload-crashes:
+	$(SENTRY_ENV_OPTIONS) MONO_PATH="$(topdir)/class/lib/build-$(HOST_PLATFORM)$(PLATFORM_PATH_SEPARATOR)$$MONO_PATH" $(RUNTIME) $(topdir)/class/lib/build-$(HOST_PLATFORM)/upload-to-sentry.exe
```

Comment:
instead of `$(topdir)/class/lib/build-$(HOST_PLATFORM)` you can use `$(topdir)/class/lib/$(BUILD_TOOLS_PROFILE) `